packages/svelte/src/compiler/phases/3-transform/client/visitors/shared/utils.js

```diff
@@ -32,6 +33,34 @@ export function get_states_and_calls(values) {
 	return { states, calls };
 }
 
+/**
+ * Escape the html in every quasi in the template literal
+ * @param {SvelteNode} template
+ * @param {boolean} [is_attr]
+ */
+export function escape_template_quasis(template, is_attr) {
+	walk(
+		template,
+		{},
+		{
+			TemplateLiteral(node, { next }) {
+				for (let quasi of node.quasis) {
+					quasi.value.raw = escape_html(quasi.value.raw, is_attr);
+				}
+				next();
+			},
+			Literal(node, { next }) {
+				if (node.raw != null) {
+					node.raw = escape_html(node.raw, is_attr);
+				} else {
+					node.value = escape_html(node.value, is_attr);
+				}
+				next();
+			}
```

Comment:
The escaping [needs to happen at runtime](https://svelte.dev/playground/hello-world?version=pr-14269#H4sIAAAAAAAAE23NMQ7CMAwF0KtEXrogupc0EucgDKU1IlLqRIlbgaLcnRgY2ayn7_8L0LQiDBBCzHCAu_OYYbgU4FcUF2j-S51jPOYdPYvdpoz_fA7ESNxqQOc5ucjqQ08eLaxh2TxaMJYsN86sZFqNqtOPZLqTJd1_vyRCenG7KRKpupfbUpuQMhg4bViv9Q3ti_IKxAAAAA==), not here